benches/oneshot/ipa.rs

```diff
@@ -45,21 +65,17 @@ async fn main() -> Result<(), Error> {
     let start = Instant::now();
     let result = world
         .semi_honest(records, |ctx, input_rows| async move {
+            let query = ipa::Query::new(&input_rows, PER_USER_CAP, MAX_BREAKDOWN_KEY)
+                .with_match_key_bits(match_key_bits);
             // ipa_wip_malicious::<Fp32BitPrime, MatchKey, BreakdownKey>(
-            ipa::<Fp32BitPrime, MatchKey, BreakdownKey>(
-                ctx,
-                &input_rows,
-                PER_USER_CAP,
-                MAX_BREAKDOWN_KEY,
-                NUM_MULTI_BITS,
-            )
-            .await
-            .unwrap()
+            ipa(ctx, query).await.unwrap()
```

Comment:
I think that we should keep two inputs here.  One for the query configuration stuff (how many bits from the match key to process, etc...) and one for all of the inputs.

If you think about how you might arrange this in a real deployment, the configuration will be shared and likely provided to the second and third helpers by the first helper, after receiving it from the record collector.  The inputs will be provided separately.  On that basis, it makes sense to keep the two separate.